pandas/core/frame.py

```diff
@@ -839,13 +852,18 @@ def to_gbq(self, destination_table, project_id=None, chunksize=10000,
         reauth : boolean (default False)
             Force Google BigQuery to reauthenticate the user. This is useful
             if multiple accounts are used.
+        if_exists : {'fail', 'replace', 'append'}, default 'fail'
+                - fail: If table exists, do nothing.
```

Comment:
alignment, text should start at same level as the explanation for the other parameters (so 4 spaces behind 'if_exists')